var/spack/repos/builtin/packages/py-biom-format/package.py

```diff
@@ -46,3 +46,4 @@ class PyBiomFormat(PythonPackage):
     depends_on('py-scipy@0.13.0:', type=('build', 'run'))
     depends_on('py-pandas@0.19.2:', type=('build', 'run'))
     depends_on('py-six@1.10.0:', type=('build', 'run'))
+    depends_on('py-pyqi', type=('build', 'run'))
```

Comment:
In the `setup.py`, I'm seeing:
```python
# HACK: for backward-compatibility with QIIME 1.9.x, pyqi must be installed.    
# pyqi is not used anymore in this project.                                     
if sys.version_info[0] < 3:                                                     
    install_requires.append("pyqi")          
```
I'm not sure exactly what it means by "pyqi is not used anymore in this project". But it seems to be a Python 2-only dependency. Last I can recall, this spec logic was broken, but you might want to add a commented out line above it like:
```python
# This dependency breaks concretization                                     
# See https://github.com/spack/spack/issues/2793
# depends_on('py-pyqi', type=('build', 'run'), when='^python@:2')
```